Dockerfile

```diff
@@ -31,6 +31,8 @@ RUN ./manage.py collectstatic --noinput
 RUN apt-get clean && apt-get purge
 USER mitodl
 
+RUN mkdir /var/media && chown -R mitodl:mitodl /var/media
```

Comment:
I'm not sure why there would be a difference (not familiar with the state of the art with OS X Docker) but it seems like moving `USER mitodl` after this line would fix the problem.